src/common/ReadMoreButton/styled.js

```diff
@@ -1,24 +1,29 @@
 import styled from "styled-components";
 
 export const Button = styled.button`
-  display: block;
+  display: inline;
   border: none;
   border-radius: 5px;
-  padding: 16px 24px;
+  padding: 8px 20px;
   font-family: "Open Sans", sans-serif;

```

Comment:
Czy to nie powinien być Poppins?

Reply:
Tutaj to akurat nasz customowy przycisk, więc w sumie jest to według uznania :D